ansible_builder/main.py

```diff
@@ -190,6 +198,15 @@ def validate(self):
                 if not os.path.exists(requirement_path):
                     raise DefinitionError("Dependency file {0} does not exist.".format(requirement_path))
 
+        ee_base_image = self.get_base_image()
```

Comment:
I hate to see `get_base_image` as its own method when it's not the canonical way that the value gets set in practice. I'd prefer to lose the method and use the same `self.definition.raw.get('base_image')` here

Reply:
Makes sense!